### ISSUE TITLE
[iOS] Player is not reporting an error with adBreak, which leads to unplayable playback

### ISSUE THREAD

**Issue by Veryyapeee** [bitmovin/bitmovin-player-react-native#56]
Regarding to this PR -> https://github.com/bitmovin/bitmovin-player-react-native/pull/54

I have some issues, I was actually implementing my own bridge for React-Native and I have some issues with VAST Ads

Here is the AdBreak URL -> https://streamit-test-czamiara.herokuapp.com/adbreak

And the adBreak is broken, on native iOS application (https://github.com/bitmovin/bitmovin-player-ios-samples/tree/main/BasicAds/BasicAds), the player is reporting an "onAdError" event
But on React-Native app (my own as well as on this implementation in this PR) the adBreak is freezing and player is not reporting any error event

Android is working great.

Repro steps:

Pass adBreak url to ad config -> https://streamit-test-czamiara.herokuapp.com/adbreak
Launch app on iPhone simulator (iOS 15.5) -> also I can repro this on real device (same iOS version)
Launch playback

I'm not looking for fixing issue with ad tag itself. I'm looking for a player to report the error (like it does on Android/Native app)/ handle it out of the box (like it does on Android/Native app)

**Comment by pogist** [comment]
Hi @Veryyapeee 👋!

I tested the ad break URL you shared with both this lib and [`BasicAds`](https://github.com/bitmovin/bitmovin-player-ios-samples/tree/main/BasicAds), but none seemed to report an `AdError`.

I also tested using the latest SDK version (`3.30.0`) and the previous one (`3.28.0`) on both this lib and the sample, but it still didn’t work. So maybe this is a problem regarding the iOS Player SDK per se, not React Native (or this library).
